content/docs/setup/kubernetes/ansible-install/index.md

```diff
@@ -13,12 +13,12 @@ Instructions for the installation and configuration of Istio using Ansible.
 
 1. One of these solutions:
 
-    * [Platform setup for Minikube](/docs/setup/kubernetes/minikube)
-    * [Platform setup for Google Kubernetes Engine](/docs/setup/kubernetes/gke)
-    * [Platform setup for IBM Cloud Kubernetes Service](/docs/setup/kubernetes/ibm)
-    * [Platform setup for Openshift](/docs/setup/kubernetes/openshift)
-    * [Platform setup for Amazon Web Services](/docs/setup/kubernetes/aws)
-    * [Platform setup for Azure](/docs/setup/kubernetes/azure)
+    * [Platform setup for Minikube](/docs/setup/kubernetes/platform-setup/minikube)
```

Comment:
Remove all this and just point to /docs/setup/kubernetes/platform-setup/

Reply:
Hmm, I would prefer to go directly where I want.

Reply:
Since this is largely a matter of opinion, it can be debated in a separate PR.